### PR TITLE
Enhance(readme): update readme and add prometheus configuration in readme

### PR DESCRIPTION
**Pull request by slalwani97**
This PR adds the Prometheus configuration to scrape monitor-pv metrics and updates readme.

Signed-off-by: Sumit Lalwani sumit.lalwani97@gmail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
 # monitor-pv
 custom stats collector for OpenEBS persistent volumes (jiva, localpv)
+
```
Comment (ksatchit) [inline README.md:3]: Lets add one pre-requisites section (openebs installation, link to docs)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+To run monitor-pv in your kubernetes cluster apply the given node-exporter yaml. It will contain two containers i.e node-exporter and monitor-pv. monitor-pv will collect the metrics and will store in a file and node-exporter will read that file and expose the metrics.
```
Comment (ksatchit) [inline README.md:4]: We can convert this to a 'How to use' section and show the command "kubectl apply -f <>" and verify these pods are created step via "kubectl get pods -n openebs". This following para can be a "How it works" section: 

```suggestion
The monitor PV daemonset pods consist of two containers i.e node-exporter and monitor-pv. The monitor-pv collects the PV size and PV utilization information & places it in a file on the shared mount. The node exporter uses its text-file exporter to expose this data as metrics.
```

**Comment by ksatchit** [comment]
Thanks @slalwani97 ! Few minor comments.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,69 @@
 # monitor-pv
 custom stats collector for OpenEBS persistent volumes (jiva, localpv)
+
+### Pre-requisite
+- openebs (https://docs.openebs.io/)
+
+### How to use
+- Run `kubectl apply -f node-exporter.yaml`
```
Comment (kmova) [inline README.md:8]: @ksatchit @slalwani97 -- how about renaming this file `node-exporter.yaml` to `node-exporter-pv-metrics.yaml` ?

Comment (ksatchit) [reply README.md:8]: yes @kmova it will be clearer.

**Pull request merged**